src/extensions/report-portal-history.js

```diff
@@ -24,11 +24,20 @@ async function getLaunchHistory(extension) {
   return [];
 }
 
+function setLinkHistoryViaProperty(extension) {
+  var linkHistoryVia = "uuid";
+  if( extension.inputs.link_history_via && extension.inputs.link_history_via != '') {
+    linkHistoryVia = extension.inputs.link_history_via;
+  }
+  return linkHistoryVia;
+}
+
 function getSymbols({ target, extension, launches }) {
   const symbols = [];
   for (let i = 0; i < launches.length; i++) {
     const launch = launches[i];
-    const launch_url = `${extension.inputs.url}/ui/#${extension.inputs.project}/launches/all/${launch.uuid}`;
+    eval("var linkHistoryVia = launch." + setLinkHistoryViaProperty(extension));
+    const launch_url = `${extension.inputs.url}/ui/#${extension.inputs.project}/launches/all/${linkHistoryVia}`;
```

Comment:
Use the input variable here.
```suggestion
    const launch_url = `${extension.inputs.url}/ui/#${extension.inputs.project}/launches/all/${extension.inputs.link_history_via}`;
```

Reply:
Yes. We can do that, but that way we should always set "link_history_via" to configuration file. With my implementation, if you not set it in configuration file it will use "uuid" as default value. If you set it, you can override it.

That way we are sure that any previous version of the application will still work and also provide the ability to overcome Report Portal "bug".

What is your opinion?

Reply:
Ahh, we don't need to set the `link_history_via` in config file because we would be having a default value of `uuid` inside the `default_inputs` object - https://github.com/test-results-reporter/reporter/blob/main/src/extensions/report-portal-history.js#L95-L98

```js
const default_inputs = {
  history_depth: 5,
  title: 'Last Runs',
  link_history_via: 'uuid'
}
```

Reply:
@uDuCkV my intention was to remove the new `setLinkHistoryViaProperty` function with the above implementation.

Reply:
Sorry, I did not see that. It is much more beautiful your way. New commit is coming.